pyomo/util/report_scaling.py

```diff
@@ -184,10 +162,9 @@ def report_scaling(m: _BlockData, too_large: float = 5e4, too_small: float = 1e-
 
     if len(cons_with_large_bounds) > 0:
         s += 'The following constraints have bodies with large bounds. Please scale them.\n'
-        longest_cname = _get_longest_name(cons_with_large_bounds)
-        s += f'{"Constraint":<{longest_cname + 5}}{"LB":>12}{"UB":>12}\n'
+        s += f'{"LB":<12}{"UB":<12}Constraint\n'
         for c, (c_lb, c_ub) in cons_with_large_bounds.items():
-            s += f'{str(c):<{longest_cname + 5}}{c_lb:>12.2e}{c_ub:>12.2e}\n'
+            s += f'{c_lb:<12.2e}{c_ub:<12.2e}{str(c)}\n'
```

Comment:
Would you consider making this right and not left justified?  Looking at the reference output in the tests, it wsa a bit jarring that the "e"'s didn't line up.

Reply:
I tried, but then I have to make the names right justified, which I think means I would need to calculate the length of the variable names?

Reply:
I was thinking making LB and UB right-justified and name left-justified.  We do this with the tabular writer, too (in general, right-justified looks good, but not at the end of the line, especially when that field is really long).

Reply:
Done!